test/test_tensorboard.py

```diff
@@ -0,0 +1,454 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+from common_utils import run_tests
+
+from torch.utils.tensorboard import x2num, SummaryWriter
+import torch
+import numpy as np
+import unittest
+from torch.utils.tensorboard import summary
+from torch.utils.tensorboard.utils import _prepare_video, convert_to_HWC
+import pytest
+# from torch.utils.tensorboard import SummaryWriter
+# from .expect_reader import compare_proto
+import os
+import sys
+import matplotlib.pyplot as plt
+# from torch.utils.tensorboard import x2num
+
+
+class PyTorchNumpyTest(unittest.TestCase):
```

Comment:
Please change the class to be `class TestTensorBoardPyTorchNumpy(unittest.TestCase):`

Reply:
Also, please use the TestCase provided in `common_test`, it provides a good hygiene setUp function that we want on all tests